recipes/example/meta.yaml

```diff
@@ -1,7 +1,7 @@
 # Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Please do not modify the example `meta.yaml`. If you have suggested changes, they should be submitted in a separate pull request.

Reply:
This was marked as resolved, but I still see modifications to `example/meta.yaml`

Reply:
My mistake, I had added back in an older version, the most recent commit should have the original examples/meta.yaml